packages/devextreme/js/__internal/ui/m_progress_bar.ts

```diff
@@ -184,6 +185,10 @@ const ProgressBar = TrackBar.inherit({
       case 'statusPosition':
         this._toggleStatus(this.option('showStatus'));
         break;
+      case 'readOnly':
+        this.callBase(args);
+        this.setAria('readonly', undefined);
```

Comment:
cannot we refactor it to not manually update element's attr in 2 places?
For example override some of method implemented on editor's level?
Now attribute is set somewhere in `callBase` and then u restore it here. Cannot we just not set it?

Reply:
[ab1c9f1](https://github.com/DevExpress/DevExtreme/pull/27984/commits/ab1c9f1c45526c7aeaee0d7864c1d5a772f50a40)